isb_lib/geome_adapter.py

```diff
@@ -96,39 +170,68 @@ def listProjects(self):
     def recordsInProject(self, project_id, record_type):
         L = getLogger()
         L.debug("recordsInProject project %s", project_id)
-        url = f"{GEOME_API}records/{record_type}/json"
+        params = {"includePublic": "true", "admin": "false"}
+        url = f"{GEOME_API}projects/{project_id}/expeditions"
```

Comment:
This is the main change -- after we hit a project, iterate over all the expeditions in the project and only fetch the records for those expeditions that have mod dates after the mod date we had previously.